packages/rest/src/rest.server.ts

```diff
@@ -196,6 +198,21 @@ export class RestServer extends Context implements Server, HttpServerLike {
     };
     this._expressApp.use(cors(corsOptions));
 
+    // Place the assets router here before controllers
```

Comment:
❗️ The static router must be mounted AFTER dynamic routes. This is super important for performance. ❗️ 

In your current design, every REST API endpoint like `/products/123` is going to hit the file system to check if there is a file matching the URL, e.g. `/html/products/123`. 

Please add a test to verify what happens when there is both a LB4 route (e.g. a controller method) and a static file mapped at the same URL.

The correct order of middleware:
 - LB4 router
 - static files
 - error handler

This is BTW the same design we have in LB 3.x, it took us few iterations and lot of learning until we figured it out correctly.

Reply:
> In your current design, every REST API endpoint like /products/123 is going to hit the file system to check if there is a file matching the URL, e.g. /html/products/123.

That's not true. `/products/123` won't match `/html` and it will be skipped without touching the file system.

> The correct order of middleware:
LB4 router
static files
error handler

I'm fine with this order but the LB4 router does not pass control to `next` at the moment.

Reply:
I thought express makes a map of the routes in memory of all the files mounted for static serving? Isn't that the point of the express Router?